back-end/app.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
```

Comment:
Pleas import using `import` instead of `require`.

Reply:
it throws an error :(

Reply:
lav dzvec